client/apps/articles_list/components/listView.jsx

```diff
@@ -0,0 +1,147 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import * as Actions from 'client/actions/articlesActions'
+import { data as sd } from 'sharify'
+
+import $ from 'jquery'
+import { debounce } from 'lodash'
+import FilterSearch from 'client/components/filter_search/index.coffee'
+import request from 'superagent'
+
+require('jquery-on-infinite-scroll')
+const query = require('../query.coffee')
+const icons = () => require('../icons.jade')(...arguments)
+
+export class ArticlesListView extends Component {
+  static propTypes = {
+    actions: PropTypes.object,
+    articles: PropTypes.array,
+    published: PropTypes.bool,
+    channel: PropTypes.object
+  }
+
+  state = {
+    articles: this.props.articles || [],
```

Comment:
I like how state is handled here; rather than dispatching local concerns back to the store just manage it with local state 👍 Makes things clearer and more precise